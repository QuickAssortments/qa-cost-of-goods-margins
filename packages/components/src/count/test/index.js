/** @format */
/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import Count from '../';

describe( 'Count', () => {
	test( 'it renders correctly', () => {
		const tree = renderer.create( <Count count={ 33 } /> ).toJSON();
		expect( tree ).toMatchSnapshot();
	} );
} );
