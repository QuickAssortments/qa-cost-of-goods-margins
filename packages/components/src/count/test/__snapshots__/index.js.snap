// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Count it renders correctly 1`] = `
<span
  aria-label="Total 33"
  className="woocommerce-count"
>
  33
</span>
`;
